rp2040-hal/src/clocks/macros.rs

```diff
@@ -352,12 +354,12 @@ macro_rules! stoppable_clock {
                     // Disable clock. On clk_ref and clk_sys this does nothing,
                     // all other clocks have the ENABLE bit in the same position.
                     self.disable();
-                    if (self.frequency > 0u32.Hz()) {
+                    if self.frequency > HertzU32::from_raw(0) {
```

Comment:
Feel more explicit even if practically useless to use `Hz(0)` performance wise it should not matter since its all const afaik